src/engine/strat_engine/backstore/metadata/mda.rs

```diff
@@ -406,7 +406,7 @@ impl MDAHeader {
         // size of metadata is greater than 2^32 - 1 bytes, which is
         // unlikely.
         assert!(*self.used.bytes() as u64 <= std::usize::MAX as u64);
-        let mut data_buf = vec![0u8; *self.used.bytes() as usize];
+        let mut data_buf = vec![0u8; convert_int!(*self.used.bytes(), u64, usize)];
```

Comment:
You can remove the assert! and the comment above it.

Reply:
Resolved.